src/interpreter/symbolTable.ts

```diff
@@ -34,12 +62,32 @@ export const extendSymbolTable = (
  */
 export const getAddressFromSymbolTable = (
```

Comment:
Can `getAddressFromSymbolTable` call `getEntryFromSymbolTable` and return `result.address` instead?